packages/anomaly-detection-store/README.md

```diff
@@ -0,0 +1,55 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Anomaly Detection Store (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in very early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A Veramo anomaly detection store plugin. This plugin stores the result of the location lookup which is part of the anomaly detection plugin
+
+## Available functions
+
+- lookupLocation
+
+## Usage
+
+### Adding the plugin to an agent:
+
+```typescript
+import { AnomalyDetectionStore } from '@sphereon/ssi-sdk.anomaly-detection'
+
+const agent = createAgent<IAnomalyDetectionStore>({
+  plugins: [
+    new AnomalyDetectionStore({
+      geoIpDBPath: './GeoLite2-Country.mmdb',
+    }),
+  ],
+})
+```
+
+### Lookup a location:
+
+```typescript
+const result = await agent.lookupLocation({
```

Comment:
Agent plugin should prefix something, to ensure that a developer can actually make head or tails about the 100+ methods available in a plugin.

So geoLookupLocation makes sense

Reply:
geolocation is a single word: https://dictionary.cambridge.org/dictionary/english/geolocation. If we want to prefix the method with geo, doesn't it make more sense to call it geolocationLookup?

Reply:
yes